### PR TITLE
Implement OfflineMode tests

### PR DESCRIPTION
**Pull request by kblok**
closes #70 

Based on #125

### CODE REVIEW

**Comment by kblok** [comment]
Don't approve it yet @Meir017 . setting this to [WIP] 
I'm cleaning up some stuff here

**Comment by kblok** [comment]
Now it's ready to go @Meir017  :)

File: lib/PuppeteerSharp.Tests/Page/OfflineModeTests.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+            await Page.SetOfflineModeAsync(false);
+            var response = await Page.ReloadAsync();
+            Assert.Equal(HttpStatusCode.OK, response.Status);
+        }
+
+        [Fact]
+        public async Task ShouldEmulateNavigatorOnLine()
+        {
+            const string navigatorOnLine = "window.navigator.onLine";
```
Comment (Meir017) [inline lib/PuppeteerSharp.Tests/Page/OfflineModeTests.cs]: I know this is used 3 times in the test but it's a bit less clear this way, might be clearer to use that string on each `EvaluateExpressionAsync` call

Comment (kblok) [reply lib/PuppeteerSharp.Tests/Page/OfflineModeTests.cs]: Makes sense

Comment (kblok) [reply lib/PuppeteerSharp.Tests/Page/OfflineModeTests.cs]: Done

**Comment by PhenX** [comment]
Hi guys, sorry for the long delay, I had no time to finish the changes, glad to see it's now closed :)

**Comment by kblok** [comment]
Yeah man, it was about pushing from master and doing some polishing.

I think I said that but, even if you don't get to the final solution, these PRs helps us A LOT. Thank you!

**Comment by Meir017** [comment]
this closed #70 instead of #7

**Comment by kblok** [comment]
Ups

**Pull request merged**